sass/_layout.scss

```diff
@@ -37,6 +37,13 @@
 
 .column {}
 
+.is-2-5 {
+    width: 20.83%;
+}
+.is-offset-0-5 {
+    margin-left: 4.125%;
+}
+
```

Comment:
Tri hero doesnt match the existing grid - need some half column width classes to do it